### PR TITLE
add homework from will

### PR DESCRIPTION
**Pull request by WillHHH**

### CODE REVIEW

File: day7/will/index.css [hunk 1]
```diff
@@ -0,0 +1,682 @@
...
+  src: url("/ttnorms/light.woff2") format("woff2");
+  src: url("/ttnorms/medium.woff2") format("woff2");
+  src: url("/ttnorms/regular.woff2") format("woff2");
+  src: url("/ttnorms/bold.woff") format("woff");
+  src: url("/ttnorms/light.woff") format("woff");
+  src: url("/ttnorms/medium.woff") format("woff");
+  src: url("/ttnorms/regular.woff") format("woff");
+}
+
+@media only screen and (max-width: 767px) {
```
Comment (mxdi9i7) [inline day7/will/index.css]: remove media query, you do not need media query for mobile, because it's the default screen size

**Review by mxdi9i7 — CHANGES_REQUESTED** [review]

File: day7/will/index.css (hunk 1 above)
Comment (WillHHH) [reply day7/will/index.css]: 哦

File: day7/will/index.css [hunk 2]
```diff
@@ -0,0 +1,682 @@
...
+
+  .mobileview-container {
+    position: absolute;
+    top: 50px;
+    text-align: center;
+  }
+
+  .mobileview-container img {
+    width: 25%;
+    z-index: -1;
```
Comment (mxdi9i7) [inline day7/will/index.css]: img can be index of 0, -1 is below surface

File: day7/will/index.css [hunk 3]
```diff
@@ -0,0 +1,682 @@
...
+    display: none;
+  }
+
+  .tabletview-wrapper {
+    position: relative;
+    display: block;
+  }
+
+  .tabletview-title-container img {
+    width: 15%;
```
Comment (mxdi9i7) [inline day7/will/index.css]: use a px value instead

File: day7/will/index.css [hunk 4]
```diff
@@ -0,0 +1,682 @@
...
+    align-items: center;
+    border-bottom: 1px solid rgba(0, 0, 0, 0.1);
+  }
+
+  .tabletview-content-title-text p {
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 17px;
+    color: #000;
+    opacity: 0.5;
```
Comment (mxdi9i7) [inline day7/will/index.css]: can we use a color with alpha value, instead of a opacity?

File: day7/will/index.css [hunk 5]
```diff
@@ -0,0 +1,682 @@
...
+
+  .tabletview-content-title-text h2 {
+    font-weight: 500;
+    font-size: 24px;
+    line-height: 28px;
+    color: #000;
+  }
+
+  .tabletview-content-title-text span {
+    color: mediumseagreen;
```
Comment (mxdi9i7) [inline day7/will/index.css]: use hex value color

File: day7/will/index.html [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+    <div class="tabletview-content-container">
+      <div class="tabletview-content-title">
+        <div class="tabletview-content-title-text">
+          <p>Belmont High School</p>
+          <h2>Monday, Feb. 24 - <span>A Day</span></h2>
+        </div>
+        <button><i class="fas fa-sign-out-alt"></i>Log Out</button>
+      </div>
+      <div class="tabletview-content-list-block">
+        <div class="tabletview-content-list">
```
Comment (mxdi9i7) [inline day7/will/index.html]: you are writing this twice for tablet and laptop, try to use only 1 set of html for those

File: day7/will/index.html [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+          <h3>6</h3>
+          <div class="tabletview-content-list-text">
+            <h4>Philosophy</h4>
+            <p>8:00-9:31 • Mr. Nessi • Room 25A</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="laptop-wrapper">
```
Comment (mxdi9i7) [inline day7/will/index.html]: use only 1 set of html for tablet and laptop

File: day7/will/index.css (hunk 2 above)
Comment (WillHHH) [reply day7/will/index.css]: adjusted

File: day7/will/index.css (hunk 3 above)
Comment (WillHHH) [reply day7/will/index.css]: adjusted

File: day7/will/index.css (hunk 4 above)
Comment (WillHHH) [reply day7/will/index.css]: adjusted

File: day7/will/index.css (hunk 5 above)
Comment (WillHHH) [reply day7/will/index.css]: adjusted

File: day7/will/index.html (hunk 6 above)
Comment (WillHHH) [reply day7/will/index.html]: adjusted

File: day7/will/index.html (hunk 7 above)
Comment (WillHHH) [reply day7/will/index.html]: adjusted

**Review by mxdi9i7 — APPROVED** [review]

**Pull request merged**